docs/DeveloperGuide.md

```diff
@@ -1331,22 +1344,142 @@ testers are expected to do more *exploratory* testing.
+
+   1. Test case: `stats c\1 s\4`<br>
+      Expected: Statistics not found. Reason stated in the status message.
+
+   1. Other incorrect display stats commands to try: `stats c\1 s\a`, `stats c\1 s\x`, `...` (where x is larger than the
+      size of student list)<br>
+      Expected: Statistics not found. Error details shown in the status message.
+
+### Displaying a lesson's venue
+
+1. Displaying a lesson's venue
+
+   1. Prerequisites: List all classes using the `list-class` command. Using default tutor's pet data.
+
+   1. Test case: `display-venue c\1 l\1`<br>
+      Expected: Venue for the first lesson in the first class of the class list found. Details of the venue shown in
+      the status message.
+
+   1. Test case: `display-venue c\1 l\2`<br>
+      Expected: Venue not found. Error details shown in the status message.
+
+   1. Other incorrect display venue commands to try: `display-venue c\1 l\a`<br>
+      Expected: Venue not found. Error details shown in the status message.
+
+### Editing a class
+
+1. Editing a class
+
+   1. Prerequisites: List all classes using the `list-class` command. At least one class in the class list.
+
+   1. Test case: `edit-class 1 n\Edited Class`<br>
+      Expected: Name of the first class in the class list have been edited to `Edited Class`. Details of edited class
+      shown in the status message.
+
+   1. Test case: `edit-class`<br>
+      Expected: No class is edited. Error details shown in the status message.
+
+   1. Other incorrect edit commands to try: `edit-class 1 n\!`, `edit-class x`
+      (where x is larger than the size of class list)<br>
+      Expected: No class is edited. Error details shown in the status message.
+
+### Finding a class or attendance
+
+1. Find a class
+
+   1. Prerequisites: List all classes using the `list-class` command. Has class(es) with the name `CS2103T` but not
+      `2103T`.
+
+   1. Test case: `find-class CS2103T`<br>
+      Expected: All classes with the name `CS2103T Tutorial` is displayed.
+
+   1. Test case: `find-class 2103T`<br>
+      Expected: No class displayed.
+
+1. Find an attendance
+
+   1. Prerequisites: List all classes and students using the `list` command. Using default tutor's pet data.
+
+   1. Test case: `find-attendance c\1 l\1 s\1 w\1`<br>
+      Expected: Attendance found. Attendance details of the student shown in the status message.
+
+   1. Test case: `find-attendance c\1 l\1 s\1 w\3`<br>
+      Expected: Attendance not found. Reason stated in the status message.
+      
+   1. Test case: `find-attendance c\1 l\1 s\4 w\3`<br>
+      Expected: Attendance not found. Reason stated in the status message.
+      
+   1. Other incorrect find commands to try: `find-attendance c\1 l\1 s\4 w\54`, `find-attendance c\x l\1 s\4 w\54`
+      (where x is larger than the size of class list)<br>
+      Expected: Attendance not found. Error details shown in the status message.
+
+### Clearing Tutor's Pet
+
+1. Clearing all classes
```

Comment:
Not sure if this section should be labeled `Clearing classes in Tutor's Pet`? Or if we plan to have all the `clear` and `reset` features in this section then I think its fine to retain the section name.